optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/CoachShuttleGathering/CoachShuttleGathering-section.adoc

```diff
@@ -0,0 +1,52 @@
+[[coachShuttleGathering]]
+= Coach shuttle gathering
+:imagesdir: ../..
+
+
+[[coachShuttleGatheringProblemDescription]]
+== Problem description
+
+Transport passengers to a hub by using coaches and
+shuttles.
+
+Hard constraints:
+
+* Coach stop limit: each coach can only visit a limited
+number of stops before arriving at the hub
+
+* Shuttle capacity: a shuttle can only hold a limited
+number of passengers
+
+* Coach capacity: a coach can only hold a limited
+number of passengers
+
+* Transport time: a passenger travelling from a particular
+bus stop upon boarding a bus must arrive at the hub within
+a specified amount of time
+
+* Shuttle destination is coach or hub: a shuttle's
+destination must be a bus stop that is visited by
+a coach
+
+Soft constraints:
+
+* Shuttle setup cost: Cost of using a shuttle
+
+* Distance from previous: Minimize distance between
+consecutive stops in routes
+
+* Distance bus stop to bus destination: Minimize
+distance between the final stop in a route and the hub
+
+* Distance coach directly to destination: Minimize
+distance for coaches that go directly to the hub
+
+[[coachShuttleGatheringProblemSize]]
+== Problem size
+
+[source,options="nowrap"]
+----
+CoachShuttleGathering example has 20 road locations, 3 coaches, 6 shuttles and 12 bus stops with a search space of 10^23.
+CoachShuttleGathering public_preselection_1 has 193 road locations, 6 coaches, 333 shuttles and 144 bus stops with a search space of 10^1311.
+CoachShuttleGathering public_preselection_2 has 193 road locations, 6 coaches, 333 shuttles and 144 bus stops with a search space of 10^1311.
```

Comment:
Would you mind adding spaces after `example` to align the `has` on all 3 lines.
Also add an extra space before `20`, 2 spaces before `6` and 1 space before `12` so all numbers are right aligned like in the other docs?